tests/parsers/locate.py

```diff
@@ -34,12 +34,31 @@ def testParseFile(self):
 
     expected_event_values = {
```

Comment:
Maybe add a comment that you're testing a path with entries

Reply:
Done